tests/untldoc_test.py

```diff
@@ -756,3 +756,11 @@ def test_generate_hash():
     test_input = [[('qualifier', 'serialtitle'), ('content', 'The Bronco')]]
     hash_val = untldoc.generate_hash(test_input)
     assert hash_val == '9eff715f7ee7da9d5c2efdf075d07225'
+
+
+def test_hash_dict_to_hash_version():
+    xml_file = open('tests/metadc_complete.untl.xml', 'rb')
+    untl_object = BytesIO(xml_file.read())
+    untl_elements = untldoc.untlxml2py(untl_object)
```

Comment:
I think you can alternatively pass the UNTL file name only and it will handle the other stuff with iterparse
```suggestion
    untl_elements = untldoc.untlxml2py('tests/metadc_complete.untl.xml')
```